config/environments/test.rb

```diff
@@ -1,6 +1,4 @@
 Rails.application.configure do
-  # Settings specified here will take precedence over those in config/application.rb.
-
```

Comment:
I may take a pass and clean up alot of the extraneous comments, but not necessary for this PR.